### DISCUSSION TITLE
element-plus源码 添加新组件 docs 应该怎么配置

### DISCUSSION THREAD

**Discussion by lixiaofa** [element-plus/element-plus#11699] [Q&A] [answered]
element-plus源码 添加新组件 docs 应该怎么配置才能预览
![image](https://user-images.githubusercontent.com/23230215/220844348-7df20a3f-ec99-46f6-9ea8-bdf8ec6118ec.png)

**Comment by tolking** [comment, marked as answer]
Link [component.json](https://github.com/element-plus/element-plus/blob/dev/docs/.vitepress/crowdin/en-US/pages/component.json)

**Reply by lixiaofa** [reply]
> Link [component.json](https://github.com/element-plus/element-plus/blob/dev/docs/.vitepress/crowdin/en-US/pages/component.json)

也要更改.md 文件

**Comment by ryuhangyeong** [comment]
There is currently a PR to accept the new component. please refer to this.

https://github.com/element-plus/element-plus/pull/11653

**Reply by lixiaofa** [reply]
> There is currently a PR to accept the new component. please refer to this.
> 
> #11653

 thank you
